RucioClient.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+"""
+Encapsulates requests to Rucio API
+Requieres:
+    rucio-client
+Environment:
+    export X509_USER_PROXY=/tmp/x509up_$UID
+    export RUCIO_HOME=~/.local/
+    ${RUCIO_HOME}/rucio.cfg
+"""
+
```

Comment:
@todor-ivanov I have put the rucio.cfg in ~/.local/etc/ for cmsunified account. I think we can put this line `export RUCIO_HOME=~/.local/` in 
1. https://github.com/CMSCompOps/WmAgentScripts/blob/master/postcycle-strict.sh
2. https://github.com/CMSCompOps/WmAgentScripts/blob/master/postcycle-update.sh
3. https://github.com/CMSCompOps/WmAgentScripts/blob/master/postcycle-review-recovering.sh
4. https://github.com/CMSCompOps/WmAgentScripts/blob/master/postcycle-review-manual.sh

Generally we export PROXY and all in credentials.sh but I haven't installed rucio in all machines and I see no reason to do so anyways. So exporting this just before we call checkor which uses rucioclient is the best place as per my understanding. what do you think?
@amaltaro @z4027163

Reply:
Hi @sharad1126 :+1: This was exactly what I was having in mind. Thanks you.